lib/kuzzle/kuzzle.js

```diff
@@ -107,6 +107,11 @@ class Kuzzle extends KuzzleEventEmitter {
 
     // Kuzzle version
     this.version = require('../../package.json').version;
+
+    /**
+     * Returns true if Kuzzle has started and is ready to handle requests
+     */
+    this.onAsk('kuzzle:started:get', () => this.started);
```

Comment:
Instead of an event, wouldn't it better to have a state machine using an enum, and available directly as a property under the kuzzle object?
Could be named `kuzzle.state`.

For now I have the following states in mind: starting, running, shutting down